custom-runtime/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Runtime
```

Comment:
I agree that `custom-runtime` is not clear as possible, but still, this is an example of customized runtime and how to build it. Not directly about making use of it.  About the second this is [this tutorial](https://github.com/kyma-project/kyma/pull/13895/files#diff-2b37051e516d8327aa69a4d97778a65e90b3656d290cb0ef7940e10605554a85R1). I can point to this tutorial as you suggested in the comment below, but we should not suggest that this may be an example of overriding runtime. Let's find a better name. Maybe `Serverless custom runtime image` + point 3 with a link to the right tutorial? Or meybe just remove the tutorial and do everything in this example?

Reply:
Agree..  So lets make it`Custom serverless runtime image`